app/client/src/features/events/EventSettings/EventDetails.tsx

```diff
@@ -78,9 +78,13 @@ export function EventDetails({ fragmentRef, className }: EventDetailsProps) {
             </ResponsiveDialog>
             {entries.map(([key, value]) => (
                 <Grid item xs={12} key={key}>
-                    <Typography variant='overline'>{key}</Typography>
+                    <Typography variant='overline'>
```

Comment:
I think this will be unnecessary with reverting `SettingsMenu` -- not 100% sure.